docker/sirius/sirius-config.yaml

```diff
@@ -1,2 +1,63 @@
 plugin: openapi
 specFile: openapi.yml
+
+resources:
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-35G3"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useDonorJsonFile.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-35G4"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useDonorJsonFileNoDecision.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-35G0"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useDonorJsonFileStop.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-0000"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useCertificateProviderJsonFile.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-0001"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useCertificateProviderJsonFileNoDecision.groovy
+
+  - path: "/api/v1/digital-lpas/M-XYXY-YAGA-0002"
+    method: get
+    response:
+      statusCode: 200
+      scriptFile: useCertificateProviderJsonFileStop.groovy
```

Comment:
Should update the README "Mocks" section to explain what these different mocked LPAs are. I can see two are used for end-to-end tests but I can't see any documentation about what the others are for